docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -14,19 +14,63 @@ helpviewer_keywords:
 * [Utf8JsonReader compared to JsonTextReader](#utf8jsonreader-compared-to-jsontextreader)
 * [Utf8JsonWriter compared to JsonTextWriter](#utf8jsonwriter-compared-to-jsontextwriter)
 
+## Table of differences between Newtonsoft.Json and System.Text.Json
+
+| `Newtonsoft.Json` feature                           | `System.Text.Json` equivalent |
+|-----------------------------------------------------|-------------------------------|
+| Case-insensitive deserialization by default         | [PropertyNameCaseInsensitive global setting](#case-insensitive-deserialization) |
+| Allow comments                                      | [ReadCommentHandling global setting](#comments) |
+| Allow trailing commas                               | [AllowTrailingCommas global setting](#trailing-commas) |
+| Allow property names without quotes                 | [Not supported](#json-strings-property-names-and-string-values) |
+| Allow single quotes around string values            | [Not supported](#json-strings-property-names-and-string-values) |
+| Allow non-string JSON values for string properties  | [Not supported](#non-string-values-for-string-properties) |
+| Custom converter registration                       | [Order of precedence differs](#converter-registration-precedence) |
+| Minimal character escaping                          | [Strict character escaping, configurable](#character-escaping) |
+| Deserialize inferred type to `object` properties    | [Requires a custom converter, sample provided](#deserialization-of-object-properties) |
+| No maximum depth by default                         | [Default maximum depth 64, configurable](#maximum-depth) |
+| `DateTimeZoneHandling`, `DateFormatString` settings | [Requires a custom converter, sample provided](#specify-date-format) |
+| Deserialize strings as numbers                      | [Requires a custom converter, sample provided](#quoted-numbers) |
+| Deserialize `Dictionary` with non-string key        | [Requires a custom converter, sample provided](#dictionary-with-non-string-key) |
+| Polymorphic serialization                           | [Requires a custom converter, sample provided](#polymorphic-serialization) |
+| Polymorphic deserialization                         | [Requires a custom converter, sample provided](#polymorphic-serialization) |
+| Required properties                                 | [Requires a custom converter, sample provided](#required-properties) |
+| Deserialize JSON `null` literal to non-nullable types | [Requires a custom converter, sample provided](#deserialize-null-to-non-nullable-type) |
+| Deserialize to immutable classes and structs        | [Requires a custom converter, sample provided](#deserialize-to-immutable-classes-and-structs) |
+| `[JsonConstructor]` attribute                       | [Requires a custom converter, sample provided](#specify-constructor-to-use) |
```

Comment:
Some of these might be over-promising what is supported. For example, just reading the table, it sounds like the attribute is supported, just requires the caller to write some code as a workaround. However, the attribute itself is not supported and you can't really make it work. In some cases, where you might use the attribute, you could do something different. 

Should we be clearer and prefix with not supported, "Not supported, requires a custom converter"?

Also, in places where we say "sample provided", the sample may not address the `Newtonsoft.Json` feature 100%.

Reply:
I'll add some text at the start of the table warning that suggested workarounds may not provide 100% of the Newtonsoft.Json functionality.

The proposed red yellow green icons will help convey that idea